docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
+---
+
+## Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will cover how to get started with your next-gen platform using Bolt for Python by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+
+### What is the next-generation platform? {#next-gen}
+
+You can find all the answers to this question on Slack's <a href="https://api.slack.com/future/intro" target="_blank">official introduction page.</a>
+
+<p class="alert alert_info"><ts-icon class="ts_icon_info_circle"></ts-icon>Our next-generation platform is currently in beta. <a href="https://api.slack.com/future/survey" target="_blank">Your feedback is most welcome.</a> - all feedback will help shape the future platform experience!</p>
+
+---
+
+### Setting up the CLI {#setting-up-cli}
```

Comment:
Instead of having three separate headers for Setting Up the CLI, Install Dependencies, Accept the Beta TOS, can I propose that we have these under a single header, with three bullet points?

Reply:
Ive combined `Setting up the CLI` and `Install dependencies` in a `Setting up` section, I also moved limitations to bottom of the tutorial 🥇